optimum/habana/peft/layer.py

```diff
@@ -29,3 +31,41 @@ def GaudiLoraLayerLinearForward(self, x: torch.Tensor, *args: Any, **kwargs: Any
 
     result = result.to(previous_dtype)
     return result
+
+class LoRALinear:
```

Comment:
Why not derive LoRALinear from PEFTLinear and monkey patch PEFTLinear at init itself? That way you will not need replace_forward at all, class created will be of type LoRALinear to begin with.

Reply:
Latest peft doesn't require this special handling. so these changes are not required anymore since we moved to peft0.10.0